modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/40_dhcp.js

```diff
@@ -171,9 +171,9 @@ return baseclass.extend({
 		}, this)), E('em', _('There are no active leases')));
 
 		return E([
-			E('h3', _('Active DHCP Leases')),
+			E('h3', _('Active IPv4 DHCP Leases')),
 			table,
-			E('h3', _('Active DHCPv6 Leases')),
+			E('h3', _('Active IPv6 DHCP Leases')),
```

Comment:
It's my understanding there is no IPv6 DHCP - only DHCPv6... Still, I guess this works.